bin/ubuntu-core-initramfs

```diff
@@ -9,14 +9,25 @@ import glob
 import shutil
 
 
+# os.path.join() throws away previous components when it finds one
+# that is an absolute path - we do not want that when using --root as
+# many defaults are absolute, so we use this function to make second
+# and later paths relative.
+def path_join_make_rel_paths(path, *paths):
+    relative_paths = []
+    for p in paths:
+        relative_paths.append(p[1:] if len(p) > 0 and p[0] == '/' else p)
```

Comment:
What happens when the path is `//foo`?
Maybe it is better to use `os.path.relpath(p, '/')`

Reply:
Good point, changed now